package.json

```diff
@@ -25,6 +25,8 @@
     "supertest": "^6.3.0"
   },
   "dependencies": {
-    "express": "^4.18.2"
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "express-router": "0.0.1"
```

Comment:
Express already contains the router, and this package is not used in your code anyway. Including unmaintained libraries is a security hazard, avoid it.
```suggestion
```